### PR TITLE
[PRM-2585] Added hidden text to LSP header

### PR DESCRIPTION
**Pull request by JBlacklaws**

### CODE REVIEW

File: test/views/components/LateSubmissionPenaltySummaryCardSpec.scala [hunk 1]
```diff
@@ -740,12 +740,12 @@ class LateSubmissionPenaltySummaryCardSpec extends SpecBase with ViewBehaviours
 
       "set the correct hidden span for a lurking point with no return submitted" in {
         doc.select("a").get(0).ownText() shouldBe "Check if you can appeal"
-        doc.select("span").text() shouldBe "for penalty on late VAT return due 30 October 2069"
+        doc.select("span").get(1).text() shouldBe "for penalty on late VAT return due 30 October 2069"
```
Comment (LewisMitchell) [inline test/views/components/LateSubmissionPenaltySummaryCardSpec.scala:743]: This check is a vulnerable to fail or prove the wrong thing:
```suggestion
        doc.select("a span").get(0).text() shouldBe "for penalty on late VAT return due 30 October 2069"
```

File: test/views/components/LateSubmissionPenaltySummaryCardSpec.scala [hunk 2]
```diff
@@ -740,12 +740,12 @@ class LateSubmissionPenaltySummaryCardSpec extends SpecBase with ViewBehaviours
...
       }
 
       "set the correct hidden span for a lurking point with a return submitted" in {
         docSubmitted.select("a").get(0).ownText() shouldBe "Appeal this penalty"
-        docSubmitted.select("span").text() shouldBe "for late VAT return due on 30 October 2069"
+        docSubmitted.select("span").get(1).text() shouldBe "for late VAT return due on 30 October 2069"
```
Comment (LewisMitchell) [inline test/views/components/LateSubmissionPenaltySummaryCardSpec.scala:748]: This check is a vulnerable to fail or prove the wrong thing:
```suggestion
        docSubmitted.select("a span").get(0).text() shouldBe "for late VAT return due on 30 October 2069"
```

File: test/views/components/LateSubmissionPenaltySummaryCardSpec.scala [hunk 3]
```diff
@@ -637,7 +637,7 @@ class LateSubmissionPenaltySummaryCardSpec extends SpecBase with ViewBehaviours
       }
 
       "shows the financial heading WITHOUT point number when the point is above threshold for filing frequency and a rewording of the appeal text" in {
-        docWithFinancialLSP.select(".app-summary-card__title").get(0).text() shouldBe "£200 penalty"
+        docWithFinancialLSP.select(".app-summary-card__title").get(0).text() shouldBe "£200 penalty for late submission of VAT due on 30 October 2069"
```
Comment (LewisMitchell) [inline test/views/components/LateSubmissionPenaltySummaryCardSpec.scala:640]: May be worth having a check for the visible text and the hidden text separately:
```suggestion
        docWithFinancialLSP.select(".app-summary-card__title").get(0).ownText() shouldBe "£200 penalty"
        docWithFinancialLSP.select(".app-summary-card__title span").get(0).ownText() shouldBe "for late submission of VAT due on 30 October 2069"
```

File: it/controllers/IndexControllerISpec.scala [hunk 4]
```diff
@@ -96,6 +96,45 @@ class IndexControllerISpec extends IntegrationSpecCommonBase {
     breathingSpace = None
   )
 
+  val getPenaltyDetailsPayloadWithOverThreshold = GetPenaltyDetails(
```
Comment (LewisMitchell) [inline it/controllers/IndexControllerISpec.scala:99]: This scenario doesn't give accurate data, this is a point applied after the threshold so ideally there needs to be other points

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

**Comment by LewisMitchell** [comment]
Could you rebase as well please? And set the PR to ready for review

**Review by AndrewRButcher — APPROVED** [review]

**Pull request merged**